app/views/calendar/when_do_the_clocks_change.html.erb

```diff
@@ -5,8 +5,6 @@
 
 <%= render :partial => "calendar_head" %>
 
-<div class="govuk-width-container">
```

Comment:
Should this "govuk-width-container" class be being removed as well?

Reply:
I've checked with Andy, and yes, we do want to remove this (the root of these views should be the bit just inside the main element, and we already have a govuk-width-container outside the main elements - the wrapper).